tests/types/test_json.py

```diff
@@ -57,5 +63,7 @@ def test_select_insert(self):
                 func.toJSONString(self.table.c.x)
             ).scalar()
             self.assertEqual(json.loads(res), data)
+        except DatabaseException as e:
+            unittest.skipIf("Code: 50" in e.args, reason="unknown JSON type")
```

Comment:
Skipping because of this [error](https://github.com/xzkostyan/clickhouse-sqlalchemy/pull/357/files#r1895010585).

Reply:
Redone to skip only code 50.